mongoid.gemspec

```diff
@@ -33,17 +33,15 @@ Gem::Specification.new do |s|
   s.required_ruby_version     = ">= 2.5"
   s.required_rubygems_version = ">= 1.3.6"
 
-  if RUBY_VERSION.start_with?('2.')
-    s.add_dependency("activemodel", [">=5.1", "<6.2"])
-  elsif RUBY_VERSION.start_with?('3.')
-    s.add_dependency("activemodel", [">=6.0", "<6.2"])
-  end
+  # activemodel 7.0.0 cannot be used due to Class#descendants issue (see: https://github.com/rails/rails/pull/43951)
+  s.add_dependency("activemodel", ['>=5.1', '<7.1', '!= 7.0.0'])
```

Comment:
I think we need a comment here saying that on Ruby 3 activemodel >= 6.0 is required to retain that note but the previous conditional that is being removed in this PR probably isn't valid in the gemspec, @comandeo do you agree?

Reply:
Rails 5.x does not support Ruby 3. This is not something specific to Mongoid, so I don't see why the Mongoid gem would need to add any comment or conditional logic here.